src/js/bin/index.js

```diff
@@ -0,0 +1,266 @@
+    process.exit(1);
+}
+
+const run = () => {
+    const targetDir = path.resolve(directoryName);
+    fs.ensureDirSync(directoryName);
+
+    if (!isSafeToCreateProjectIn(targetDir, directoryName)) {
+        process.exit(1);
+    }
+
+    console.log();
+    console.log();
+    console.log('👋🏻  Hi! You ready to create a new project?');
+    promptPackageJson();
+};
+
+const prompt = question =>
+    new Promise((resolve, reject) => {
+        rl.question(question, answer => {
+            resolve(answer);
+        });
+    });
+
+const promptForPackageJson = (key, question) =>
+    new Promise((resolve, reject) => {
+        const defaultConfig = packageConfig[key] ? chalk.grey(` ${packageConfig[key]}`) : '';
+        prompt(`${question}${defaultConfig} `).then(answer => {
+            packageConfig[key] = answer || packageConfig[key];
+            resolve(packageConfig[key]);
+        });
+    });
+
+const promptPackageJson = () => {
+    console.log();
+    console.log("Just answer a couple of questions and you'll be ready to go");
+    console.log();
+
+    promptForPackageJson('name', 'package name:')
+        .then(name => {
+            if (!isNameValid(name)) {
+                process.exit(1);
+            }
+            return promptForPackageJson('version', 'version:');
+        })
+        .then(version => {
+            if (!isVersionValid(version)) {
+                process.exit(1);
+            }
+            return promptForPackageJson('description', 'description:');
+        })
+        .then(() => {
+            return promptForPackageJson('author', 'author:');
+        })
+        .then(() => {
+            return promptForPackageJson('keywords', 'keywords:');
+        })
+        .then(keywords => {
+            packageConfig.keywords = keywords ? keywords.split(/[\s,]+/) : '';
+            confirmProjectCreation(directoryName, packageConfig);
+        });
+};
+
+const confirmProjectCreation = (directoryName, packageConfig) => {
+    const question = `${JSON.stringify(packageConfig, null, 2)}\n\nIs that okay? ${chalk.grey('yes')}`;
+
+    rl.question(question, answer => {
+        if (answer) {
+            promptPackageJson();
+        } else {
+            createProject(directoryName, packageConfig);
+            rl.close();
+        }
+    });
+};
+
+const printValidationResults = results => {
+    if (typeof results !== 'undefined') {
+        results.forEach(error => {
+            console.error(chalk.red(`  *  ${error}`));
+        });
+    }
+};
+
+const shouldBeIgnored = src => {
+    for (var i = 0; i < ignoreFiles.length; i++) {
+        if (src.indexOf(ignoreFiles[i]) === 0) {
+            return true;
+        }
+    }
+    return false;
+};
+
+const createProject = (name, config) => {
+    const targetDir = path.resolve(name);
+    const sourceDir = path.resolve(__dirname, '../../../');
+    const appName = path.basename(targetDir);
+
+    console.log();
```

Comment:
Maybe replace all `console.log()` with `log()`, adding at the top of the file `const log = console.log`